enhancements/cluster-logging/forwarder-to-otlp.md

```diff
@@ -0,0 +1,204 @@
+
+* As an administrator, I want to forward logs to an OTLP receiver
+so that I can use my observability tools to evaluate all my signals (e.g. logs, traces, metrics).
+
+### Goals
+
+* Implement OTLP over HTTP using text(i.e. JSON) in the log collector to forward to any receiver that implements OTLP
+**Note:** Stretch goal to implement OTLP over HTTP using binary(i.e. protobuf)
+
+* Deprecate the [ViaQ Data Model](https://github.com/openshift/cluster-logging-operator/blob/release-5.9/docs/reference/datamodels/viaq/v1.adoc) in
+favor of [OpenTelemetry Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/)
+* Allow customer to continue to use **ClusterLogForwarder** as-is in cases where they are not ready or not interested in OpenTelementry
+
+### Non-Goals
+
+* Replace the existing log collection agent with the [OTEL Collector](https://opentelemetry.io/docs/collector/)
+* Utilize OpenTelemetry Semantic Conventions when writing to other `Output` types
+
+## Proposal
+
+This section should explain what the proposal actually is. Enumerate
+*all* of the proposed changes at a *high level*, including all of the
+components that need to be modified and how they will be
+different. Include the reason for each choice in the design and
+implementation that is proposed here.
+
+To keep this section succinct, document the details like API field
+changes, new images, and other implementation details in the
+**Implementation Details** section and record the reasons for not
+choosing alternatives in the **Alternatives** section at the end of
+the document.
+
+This proposal inte
+
+### Workflow Description
+
+**cluster administrator** is a human responsible for administering the **cluster-logging-operator**
+and **ClusterLogForwarders**
+
+1. The cluster administrator deployes the cluster-logging-operator if it is already not deployed
+1. The cluster administrator edits or creates a **ClusterLogForwarder** and defines an OTLP output
+1. The cluster administrator references the OTLP output in a pipeline
+1. The cluster-logging-operator reconciles the **ClusterLogForwarder**, generates a new collector configuration,
+and updates the collector deployment
+
+
+### API Extensions
+
+```yaml
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+outputs:
+- name:
+  type:           # add otlp to the enum
+  tls:
+    secret:             #the default resource to search for keys
+      name:                # the name of resource
+      cacert:
+        secret:               #enum: secret, configmap 
+          name:                # the name of resource
+        key:                 #the key in the resource
+      cert:
+        key:                 #the key in the resource
+      key:
+        key:                 #the key in the resource
+    insecureSkipVerify:
+    securityProfile: 
+  otlp:
+    url:                   #must terminate with '/v1/logs'
```

Comment:
We don't need a second URL here, we can use the generic URL field in the output.

Reply:
> We don't need a second URL here, we can use the generic URL field in the output.
There is no generic URL field in the "next" api; it is output specific